Makefile

```diff
@@ -91,3 +91,19 @@ github-build-win:
 workflows:
 	cargo run -p mk-workflows
 
+# Tests local builds based on the env vars `SKIA_BUILD_DEFINES` and `SKIA_LIBRARY_SEARCH_PATH`.
+# 
+# This builds first a set of libraries, copies them away and then tries to build with the libraries
+# defined externally.
+#
+# https://github.com/rust-skia/rust-skia/pull/527
+
+.PHONY: test-local-build
+test-local-build:
+	cargo clean
+	cargo build --release
+	rm -rf tmp/
+	mkdir -p tmp/
+	find target -name "libskia*.a" -type f -exec cp {} tmp/ \;
```

Comment:
If this doesn't work try `target/debug/build/skia-bindings-*/out/skia/*.a` I'm not sure which specifically are needed but they might not all match your pattern, this is what's in my `out/skia`:

 * libfreetype2.a
 * libpng.a
 * libskparagraph.a
 * libbrotli.a
 * libharfbuzz.a
 * libskia-bindings.a
 * libskshaper.a
 * libexpat.a
 * libicu.a
 * libskia.a
 * libzlib.a

Reply:
From what I know `libskia.a` will contain all the third party object files. With the feature `textlayout` enabled, `libskshaper.a` and `libskparagraph.a` are needed, but there is also a problem with a missing `icudtl.dat` that is expected to be in `OUT_DIR/skia`. I'll take a look at that later.